lesson15/main.js

```diff
@@ -72,9 +72,10 @@ const init = async (val) => {
   createNewList(listsData);
 };
 
-const getNumbers = () => {
+const inputValue = () => {
```

Comment:
IMO

function name is usually to use `verb + noun.
hint
If you look closely, it is a function that processes and returns characters.

Reply:
Thank you for the advice and hint. I changed `returnInputValue`. 
Dose it make sense?

[1957e93](https://github.com/yuka830/js-lessons/pull/17/commits/1957e93691508eddda65472eb545f22268f1c89d)

Reply:
IMO

```js
const getInputValues = () => {
  const inputNumber = document.getElementById("js-number");
  const inputName = document.getElementById("js-name");
  return { number: inputNumber.value, name: inputName.value };
};


`returnInputValue` 

Return is rarely used in function names
Because the function returns a value
Your theory is that every function name has a return

get is often used

Also, it is returned as a String value in the function,
It will be the data to send to the server, so you should return the object
I don't often send strings as they are

Reply:
I understand. You're right, everything returns.
This time I made new variable to store of string data and renamed this function to get it.
How about this?
[7f21f76](https://github.com/yuka830/js-lessons/pull/17/commits/7f21f76ea6ff408fbf705828890360c0b75c616c)

Reply:
Thank you for your time and concrete explanation.
I renamed it from `getInputStringValue` to `getNumberAndNameString`. I learned giving specific name makes it easier for others to understand and help prevent errors this lesson.
I'm appreciated.

[21b3554](https://github.com/yuka830/js-lessons/pull/17/commits/21b3554c49f9e79b25b28d8314e290d3ee2cdcfa)